.github/workflows/run-archiver.yml

```diff
@@ -68,7 +68,13 @@ jobs:
           # ZENODO_TOKEN_UPLOAD: ${{ secrets.ZENODO_TOKEN_UPLOAD }}
           # ZENODO_TOKEN_PUBLISH: ${{ secrets.ZENODO_TOKEN_PUBLISH }}
         run: |
-          conda run -n pudl-cataloger pudl_archiver --sandbox --datasets ${{ matrix.dataset }}
+          conda run -n pudl-cataloger pudl_archiver --sandbox --datasets ${{ matrix.dataset }} --sumary-file ${{ matrix.dataset }}_run_summary.json
```

Comment:
nit: this flag should probably be called `--summary-file`

Reply:
Fixed